src/controllers/legacycontrollersettingslayout.cpp

```diff
@@ -53,13 +54,18 @@ QWidget* LegacyControllerSettingsLayoutContainer::build(QWidget* pParent) {
 }
 
 QWidget* LegacyControllerSettingsGroup::build(QWidget* pParent) {
-    auto pContainer = make_parented<QGroupBox>(m_label, pParent);
+    auto pContainer = make_parented<WCollapsibleGroupBox>(m_label, pParent);
```

Comment:
wdyt of making this configurable in the settings XML? Shouldn't be hard IMO.

Reply:
Do you mean with an extra xml node?
If yes, I 'd rather not put that work on mapping developers. I mean collapsing top-level groups is desirable, and on lower levels it only makes the GUI more complex.

But maybe we can check if the groupbox' parent is a WCollapsibleGroupBox, too, and auto-set checkable if it's not.
Will check..

Reply:
Naa, we already query the groups in DlgPrefController so that's the place to set the checkable IMO.

Reply:
> Do you mean with an extra xml node?

Either that or add an attribute to the node

> If yes, I 'd rather not put that work on mapping developers. 

Right. On the flipside (and why I was suggesting to make it configurable), what if devs really want some settings to be easily visible and hide others (more advances ones for instance)?

Reply:
Ah okay, so you're proposing to allow setting the default `collapsed` state in xml (not the collapsible feature in general).

Reply:
yeah, that works too.

Reply:
I like the idea of the `collapsed` default state! I was looking at adding some kin d of `depends` or `condition` to help with the large amount of settings in #13653, but this looks like anm elegant solution to address this with a minimum effort.

Reply:
Thanks for the feedback!

Though I'd love to merge the basic implementation first.